js/components/scrubberBar.js

```diff
@@ -36,7 +37,7 @@ var ScrubberBar = React.createClass({
   },
 
   componentWillReceiveProps: function(nextProps) {
-    if (this.transitionedDuringSeek && !nextProps.seeking) {
+    if (this.state.transitionedDuringSeek && !nextProps.seeking) {
```

Comment:
FYI @chibzu this could have been a possible bug or 'fixing' it could introduce a bug.

After update, I did not notice any bugs while testing.